tools/bumper/cnao_repo_commands.go

```diff
@@ -212,6 +215,138 @@ func (cnaoRepoOps *gitCnaoRepo) collectModifiedToTreeList(allowedList []string)
 	return entries, nil
 }
 
+// getBumpBranch creates a new bump branch.
+// Since we don't want to use already open branch
+// that may hold outdated/needs rebasing changes
+// we concat a random string to the end of the branch name
+func (cnaoRepoOps *gitCnaoRepo) getBumpBranch(prTitle string) (*github.Reference, string, error) {
+	branchBaseName := strings.Replace(strings.ToLower(prTitle), " ", "_", -1)
+	branchOpenMaxTries := 10
+	for branchIdx := 0; branchIdx < branchOpenMaxTries; branchIdx++ {
+		branchNameWithRandSuffix := fmt.Sprintf("%s_%s", branchBaseName, rand.String(5))
+		branchRef, _, err := cnaoRepoOps.githubInterface.GetBranchRef(cnaoRepoOps.getCnaoRepoOwnerFromUrl(), cnaoRepoOps.getCnaoRepoNameFromUrl(), "refs/heads/"+branchNameWithRandSuffix)
+		if err != nil {
+			if cnaoRepoOps.branchNotFound(branchRef, err) {
+				return cnaoRepoOps.createNewGithubBranch(branchNameWithRandSuffix)
+			} else {
+				return nil, "", errors.Wrap(err, "Failed to check if branch is already open")
+			}
+		}
+
```

Comment:
Add some sleep to give some air to the network or whatever is failing.

Reply:
removed the iterating in favor of random seed to make sure branch name is unique.

if there is some github downtime I prefer it would exit the script, don't you agree?

Reply:
agree, closing this.